internal/cmd/run/testcafe.go

```diff
@@ -144,7 +146,7 @@ func runTestcafeInDocker(p testcafe.Project, testco testcomposer.Client, rs rest
 	log.Info().Msg("Running Testcafe in Docker")
 	printTestEnv("docker")
 
-	cd, err := docker.NewTestcafe(p, &testco, &testco, &rs, &rs, createReporters(p.Reporters))
+	cd, err := docker.NewTestcafe(p, region.None, &testco, &testco, &testco, &rs, &rs, createReporters(p.Reporters))
```

Comment:
Why is `region.None` being passed in? This seems wrong, as the region can never be None beyond the validation point.

Reply:
I was using regio region.Region for building URL, but URL now comes in a proper format and I forgot to delete region variable. I fixed this issue.